backend/services/goals_service.py

```
@@ -231,3 +238,116 @@ def check_and_populate_goals(self, user_id: int):
             self.populate_monthly_goals(user_id)
 
         logger.info("New goals populated successfully")
+
+    def initialize_user_goals(self, user_id: int):
+        # Used when signs up for the first time to account for sign up date
+        # being after the first of the month
+        self.populate_daily_goals(user_id)
+        self.populate_monthly_goals(user_id)
+
+    @staticmethod
+    def add_personalized_goal(user_id, time_period, type, requirement):
```

Ruff (0.8.2)

Missing return type annotation for staticmethod `add_personalized_goal`

(ANN205)